pretix_sepadebit/templates/pretix_sepadebit/export.html

```diff
@@ -83,6 +84,9 @@ <h2>{% trans "Exported XML files" %}</h2>
                             <span class="label label-warning">{% trans "TEST MODE" %}</span>
                         {% endif %}
                     </td>
+                    <td>
+                        {{ export.collection_date|date:"SHORT_DATE_FORMAT" }}
```

Comment:
Don't you think this will be confusing with an empty column for files containing multiple collection dates (mode = mix)?